doc/userguide/src/SupportingTools/Libdoc.rst

```diff
@@ -220,16 +225,36 @@ Robot Framework 4.0 introduced new spec version `3`.
 
 .. note:: Robot Framework 4.0 introduced an updated xml format. It is
           incompatible with former versions. LibDoc can only read and write
-          new XML format.
+          new XML format version 3.
 
-          The `xml:html` format introduced in Robot Framework 3.2. has been
-          replaced by the option `-s --specdocformat <raw|html>`.
+          The `XML:HTML` format introduced in Robot Framework 3.2. has been
+          replaced by the format `LIBSPEC` ot the option combination
```

Comment:
- Do we really have `LIBSPEC` format? Isn't it only an extension?
- "ot" -> "or"

Reply:
There was this extra special handling of `extension == libspec` then format is `XML:HTML`
That would have to be replace by `*.libspec` would be `format = XML` and `specdocformat = HTML` except the specdocformat was given...

to remove that and create consistency it was imho the best way to just say that when the extension `*.libspec` is allowed, why not just have it as a format as well.